content/docs/1.6.4/deploy/important-notes/index.md

```diff
@@ -224,14 +224,18 @@ Longhorn v{{< current-version >}} does not allow the following:
 However, Longhorn still allows the listed actions when cloning or restoring volumes created with earlier Longhorn
 versions.
 
-### Backup Related Custom Resources Might Disappear
+### Backup Data On The Remote Backup Server Might Be Deleted
 
-Longhorn will attempt to clean up the backup-related custom resources in the following scenarios:
+Longhorn may unintentionally delete backup-related custom resources (such as `BackupVolume`, `BackupBackingImage`, `SystemBackup`, and `Backup`) and backup data on the remote backup server before Longhorn v{{< current-version >}} in the following scenarios:
 
 - An empty response from the NFS server due to a brief server downtime.
-- A race condition between related Longhorn backup controllers.
+- A race condition could delete the remote backup volume and its corresponding backups when the backup target is reset within a short period.
+
+Starting with Longhorn v{{< current-version >}}:
+
+- Longhorn only deletes the backup-related custom resources in the cluster and does not delete anything on the remote backup server if there is a discrepancy between the backup information in the cluster and that on the remote backup server.
+- The backup-related custom resources in the cluster may be deleted unintentionally while the remote backup data remains safely stored. These resourcres will be resynchronized from the remote backup server during the next polling interval once the backup target is available.
```

Comment:
```suggestion
Starting with v{{< current-version >}}, Longhorn handles backup-related custom resources in the following manner:

- If there are discrepancies between the backup information in the cluster and on the remote backup server, Longhorn deletes only the backup-related custom resources in the cluster. 
- The backup-related custom resources in the cluster may be deleted unintentionally while the remote backup data remains safely stored. The deleted resources are resynchronized from the remote backup server during the next polling period (if the backup target is available).
```

Reply:
I replaced "interval" with "period" because that seems to be the appropriate word for your use case. "interval" refers to the time between polling activities.

Reply:
Modified. Thanks.